com.unity.shadergraph/Editor/Drawing/Blackboard/BlackboardFieldKeywordView.cs

```diff
@@ -132,9 +137,9 @@ private void AddCallbacks()
             {
                 int indent = 14;
                 var displayRect = new Rect(rect.x + indent, rect.y, (rect.width - indent) / 2, rect.height);
-                EditorGUI.LabelField(displayRect, "Display Name");
+                EditorGUI.LabelField(displayRect, "Entry Name");
```

Comment:
Why was this name change made here? If its clearer as to its purpose, I would also want the various `display` variables to also be updated

Reply:
It was changed to clearer as "display" vs "reference" in properties are otherwise allowed to be different from each other, but in this case it doesn't work properly. So I'm trying to differentiate for the user what this actually represents

Reply:
so is it something that should be reflected in the variable names as well, or do you not think it matters?

Reply:
I think the variable names still make sense as is but it's not a hill i'll die on